pkg/storage/chunk/cache/groupcache.go

```diff
@@ -243,7 +245,7 @@ func (c *group) Store(ctx context.Context, keys []string, values [][]byte) error
 
 	var lastErr error
 	for i, key := range keys {
-		if err := c.cache.Set(ctx, key, values[i], time.Time{}, false); err != nil {
+		if err := c.cache.Set(ctx, key, values[i], time.Time{}, c.hotCacheEnabled); err != nil {
```

Comment:
I think the simpler way here will be to check if `c.GetCacheType() != stats.ChunkCache`

Reply:
thought about that but. But would be easier if we want to experiment it with other types of cache as well. Also to benchmark same cache with and without hotcache for example.

Reply:
Let's start simple, and make it configurable if we need to.

The config option `hotCacheEnabled` is not really accurate because hot cache is still enabled either way, this setting here just controls if we write directly into hot cache

Reply:
Sure. updated.